.travis.yml

```diff
@@ -85,110 +54,41 @@ cache:
     - node_modules
     - $HOME/.atom
     - $HOME/.stack
+    - $HOME/.opam
 
 before_install:
-  # Install Homebrew on Linux
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-      git clone --depth=1 https://github.com/Linuxbrew/brew.git ~/.linuxbrew || true;
-    fi
-  # Update Homebrew
-  - brew update
-  - brew tap homebrew/dupes
-  - brew tap homebrew/versions
-  # Ruby language support
-  # - gem install ruby-beautify --verbose
-  # - gem install rubocop
-  - gem install htmlbeautifier
-  - gem install puppet-lint
-  # Sass language support
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      gem install sass;
-    else
-      docker pull unibeautify/sass-convert;
-    fi
-  # Python language support
+    # Linux: Install Nix and Packages, Python install
+    # OSX: Update Homebrew, remove conflicting cask, brew bundle, symlink elm-format
```

Comment:
How about moving these comments closer to where these things happen?
Or remove them entirely. I don't want to see them become out of date in future commits 😉 .

Reply:
super hard to do with the `if` statements, `travis` has a really weird/slightly undocumented way of interpreting/escape `bash` in it's `yaml`. see https://travis-ci.org/Glavin001/atom-beautify/builds/327062135

do you want to separate that script out into a seperate file?